content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,155 @@ aliases:
+along with your services. All traffic that your mesh services send and receive
+(data plane traffic) is proxied through Envoy, making it easy to direct and
+control traffic around your mesh without making any changes to your services.
+
+If you’re interested in the details of how the features described in this guide
+work, you can find out more about Istio’s traffic management architecture in the
+[Architecture](#architecture) section at the end of this document. The rest of
+this guide introduces Istio’s traffic management features.
+
+## Introducing Istio Traffic Management
+
+In order to direct traffic around your mesh, Istio needs to know where all your
+endpoints are, and which services they belong to. To populate its own service
+registry, Istio connects to a service discovery system. For example, if you've
+installed Istio on a Kubernetes cluster, then by default Istio knows about all the
+services and endpoints in that cluster.
+
+Using this service registry, the Envoy proxies can then direct traffic to the
+relevant services. Most microservice-based applications have multiple instances
+of each service workload to handle service traffic, sometimes referred to as a
+load balancing pool. By default, the Envoy proxies distribute traffic across
+each service’s load balancing pool using a round-robin model, where requests are
+sent to each pool member in turn, returning to the top of the pool once each
+service instance has received a request.
+
+While this basic service discovery and load balancing gives you a working
+service mesh, it’s far from all that Istio can do. In many cases you might want
+more fine-grained control over what happens to your mesh traffic. For example,
+you might want to direct a particular percentage of traffic to a new version of
+a service as part of A/B testing, or apply a different load balancing policy to
+traffic for a particular subset of service instances. You might also want to
+apply special rules to traffic coming into or out of your mesh, or add an
+external dependency of your mesh to the service registry. You can do all this
+and more by adding your own traffic configuration to Istio using Istio’s traffic
+management API.
+
+Like other Istio configuration, the API is implemented as Kubernetes custom
+resources (CRDs), which you can configure using YAML, as you’ll see in the
+examples.
+
+The rest of this guide looks at each of the traffic management API’s resources
+and what you can do with them. These resources are:
+
+-   [Virtual services](#virtual-services)
+-   [Destination rules](#destination-rules)
+-   [Gateways](#gateways)
+-   [Service entries](#service-entries)
+-   [Sidecars](#sidecars)
+
+This guide also gives an overview of some of the
+[network resilience and testing features](#network-resilience-and-testing) that
+are built in to the API’s resources.
+
+## Virtual services {#virtual-services}
+
+[Virtual services](/docs/reference/config/networking/v1alpha3/virtual-service/),
```

Comment:
Maybe it would be better to reference the actual resources in the ref guide, rather than another overview.
```suggestion
[Virtual services](/docs/reference/config/networking/v1alpha3/virtual-service/#VirtualService),
```